example/geojson/WaiteOvalTest1.geojson

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Lets just standardise the tab size for this file to the rest of the STAC for readability purposes, so two spaces it looks like.

Reply:
I am not changing this for now as this is why the CI/CD is failing (linting errors)